### ISSUE TITLE
PC8300 FPGA progress?

### ISSUE THREAD

**Issue by bance** [zzemu-cn/LASER310_FPGA#4]
Hi! can't believe what I found here - great work! my first computer as a child was a NF300:). it's mentioned in your laser310 fpga documents something was going on with PC8300 - wondering how it went? really wish I could recreate the machine on FPGA and play the classic game DaMao again.

Thanks for your great work!
bance

**Comment by yuanb** [comment]
Hi,

Thanks for you message. Lambda 8300(PC-8300) is a improved version of Sinclair ZX81 through itsROM is not entirely compatible with ZX81.

**Comment by bance** [comment]
Exactly. the C4005/C4335 chip in lambda is a bit different to the ULA in ZX81, e.g. it contains fonts data for video generation. That's why I was excited while reading your plan on PC8300 FPGA implementation. Are you still working on the PC8300 project?

All the best,

**Comment by yuanb** [comment]
No, I'm not working on it. I'm here helping zzemu to translate and organize the project. zzemu is the author. I can ask him

**Comment by bance** [comment]
I see. thought you were behind zzemu's projects. thanks for passing the words to zzemu.

btw, does zzemu have a website documenting his/her amazing works? tried google but nothing showed up.

**Comment by yuanb** [comment]
I asked zzemu, he said he might be able to build one, there is no timeline though.
Stay tuned

**Comment by bance** [comment]
That's great news! thank you.

I was trying to patch a zx81 fpga implementation to lambda pc8300, but got stuck due to lack of knowledge of 8300's ULA.  The reason I'm particularly interested in 8300 is my first computer NF300 basically shares the same hardware running different rom. I tested the dumped rom with lambda software emulator and works fine.

**Comment by yonglak** [comment]
Hi,
Wonderful projects. 
Just to check about the COLOR forgnd,backgnd command:

COLOR 4,0  - backgnd should change to green
COLOR 4,1 - backgnd should change to orange

The backgnd did not change on this core. Just for your info and (perhaps correction?)

..lak from Singapore
